cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/DistributedStreamService.java

```diff
@@ -534,9 +551,8 @@ protected StreamSizeAggregator(Id.Stream streamId, long baseCount, int streamThr
       this.cancellable = cancellable;
       this.isInit = true;
       this.streamFeed = new Id.NotificationFeed.Builder()
-        .setNamespaceId(streamId.getNamespaceId())
-        .setCategory(Constants.Notification.Stream.STREAM_FEED_CATEGORY)
-        .setName(streamId.getName())
+        .setNamespaceId(streamId.getNamespaceId()).setCategory(Constants.Notification.Stream.STREAM_FEED_CATEGORY)
+        .setName(String.format("%sSize", streamId.getName()))
```

Comment:
That's probably a better option, yes, to have a ``streamSize`` category. What do you think @chtyim ?